src/main/python/main.py

```diff
@@ -59,16 +59,17 @@ def question_style(self):
 
     @cached_property
     def credential_store(self):
-        store = None
-        file = ".credential_store"
+        store_name: str = "store"
         try:
-            store = self.get_resource(file)
+            # Get abs path to the store; platform independent
+            store: str = self.get_resource(store_name)
         except FileNotFoundError:
-            # create file
-            with open(path.join(self.get_resource(), file), "w+"):
-                pass
-        finally:        
-            return store if store else self.get_resource(file)
+            # TODO: At this point we cannot create the file so we'll need to handle this somehow
+            store: str = "store"
```

Comment:
this is the place I'm referring to in my comment - why not make this a "dict" encoded as json e.g. `json.dumps({'token': 'invalid'})`?

Reply:
Ah I see. This function is actually just grabbing the path to the `store`. We're using `get_resource` here as a way to return a platform aware path (i.e. /path/on/mac or C:\Im\a\dumb\windows\path). So if we hit the except block we're really just giving a default path of `"./store"`.

As an aside, this should probably be renamed to something a bit more indicative of what it's actually doing.

Reply:
gotcha